scenarios/secrets_in_the_cloud/terraform/user_data.tpl

```diff
@@ -142,3 +137,13 @@ vault kv put secret/tylers_seekrit value='TylerTantalizingTacosTangleToucans'
 vault kv put secret/brads_seekrit value='BradBefriendsBouncingBlueberryBison'
 export SSH_PRIVATE_KEY="${private_key}"
 vault kv put secret/id_rsa value="$SSH_PRIVATE_KEY"
+
+# Update MOTD with a hint
```

Comment:
What is MOTD?

Reply:
MOTD stand for message of the day. long ago I think when people had to dial into different message boards, basically they just were connecting to different servers, and each server had their own MOTD when you logged in to greet you. That's just one example, but I think there were a lot of other user cases: https://en.wikipedia.org/wiki/Message_of_the_day